python/ray/dag/compiled_dag_node.py

```diff
@@ -2289,16 +2618,30 @@ def visualize(
 
         Args:
             filename: The name of the output file (without extension).
-            format: The format of the output file (e.g., 'png', 'pdf').
-            view: Whether to open the file with the default viewer.
+            format: The format of the output file (e.g., 'png', 'pdf', 'ascii').
+            view: For non-ascii: Whether to open the file with the default viewer.
+                For ascii: Whether to print the visualization and return None
+                    or return the ascii visualization string directly.
             return_dot: If True, returns the DOT source as a string instead of figure.
```

Comment:
Can you unify the behavior here? We can remove this param and always return:
- for graphviz, return the dot string
- for ascii, return the ascii string

Reply:
fixed!